upgrade/helpers/tasks.py

```diff
@@ -853,3 +854,23 @@ def add_custom_product_subscription_to_hosts(product, hosts):
             host = entities.Host().search(query={'search': 'name={}'.format(host)})[0]
             entities.HostSubscription(host=host).add_subscriptions(
                 data={'subscriptions': [{'id': sub.id, 'quantity': 1}]})
+
+
+def check_status_of_running_task():
```

Comment:
@devendra104 Can you please make it more generic like:
`check_status_of_running_task(command, attempt)`